ion-schema/src/result/mod.rs

```diff
@@ -94,18 +108,29 @@ pub fn unresolvable_schema_error_raw<S: AsRef<str>>(description: S) -> IonSchema
     }
 }
 
+/// Creates an [`IonSchemaError::InvalidSchemaError`]
+macro_rules! invalid_schema {
+    ($($schema_id:ident, $($loc:ident,)?)? $fmt_string:literal $($args:tt)*) => {
+        Result::Err(
+            crate::result::IonSchemaError::InvalidSchemaError(crate::result::InvalidSchemaError::builder(format!($fmt_string $($args)*))
+                $(.with_schema_id($schema_id)
+                $(.with_isl_source_location($loc))?)?
+                .build()
+        ))
+    };
+}
+pub(crate) use invalid_schema;
+
 /// A macro that checks some condition required to be valid ISL.
 ///
 /// If invalid, returns an InvalidSchemaErr with the given error message.
-#[macro_export]
```

Comment:
❗ This internal-only macro was inappropriately being exported by the crate. I've replaced the `#[macro_export]` with a `pub(crate) use` instead.